go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/ory/dockertest/v3 v3.9.1
 	github.com/osmosis-labs/bech32-ibc v0.3.0-rc1
+	github.com/osmosis-labs/osmosis/v7 v7.3.0
```

Comment:
This was done wrong -- scanning for this was hard due to not separating out the backport vs the patch to make ti work!

Reply:
I fixed all import paths but this was left unused - thanks for catching that.